src/referencePackages/src/system.text.json/8.0.5/lib/netstandard2.0/System.Text.Json.cs

```diff
@@ -0,0 +1,2095 @@
+
+        public sealed override bool CanConvert(Type typeToConvert) { throw null; }
+
+        public sealed override JsonConverter CreateConverter(Type typeToConvert, JsonSerializerOptions options) { throw null; }
+    }
+
+    public partial class JsonStringEnumConverter<TEnum> : JsonConverterFactory where TEnum : struct, Enum
+    {
+        public JsonStringEnumConverter() { }
+
+        public JsonStringEnumConverter(JsonNamingPolicy? namingPolicy = null, bool allowIntegerValues = true) { }
+
+        public sealed override bool CanConvert(Type typeToConvert) { throw null; }
+
+        public sealed override JsonConverter? CreateConverter(Type typeToConvert, JsonSerializerOptions options) { throw null; }
+    }
+
+    public enum JsonUnknownDerivedTypeHandling
+    {
+        FailSerialization = 0,
+        FallBackToBaseType = 1,
+        FallBackToNearestAncestor = 2
+    }
+
+    public enum JsonUnknownTypeHandling
+    {
+        JsonElement = 0,
+        JsonNode = 1
+    }
+
+    public enum JsonUnmappedMemberHandling
+    {
+        Skip = 0,
+        Disallow = 1
+    }
+
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Interface, AllowMultiple = false, Inherited = false)]
+    public partial class JsonUnmappedMemberHandlingAttribute : JsonAttribute
+    {
+        public JsonUnmappedMemberHandlingAttribute(JsonUnmappedMemberHandling unmappedMemberHandling) { }
+
+        public JsonUnmappedMemberHandling UnmappedMemberHandling { get { throw null; } }
+    }
+
+    public abstract partial class ReferenceHandler
+    {
+        public static ReferenceHandler IgnoreCycles { get { throw null; } }
+
+        public static ReferenceHandler Preserve { get { throw null; } }
+
+        public abstract ReferenceResolver CreateResolver();
+    }
+
+    public sealed partial class ReferenceHandler<T> : ReferenceHandler where T : ReferenceResolver, new()
+    {
+        public override ReferenceResolver CreateResolver() { throw null; }
+    }
+
+    public abstract partial class ReferenceResolver
+    {
+        public abstract void AddReference(string referenceId, object value);
+        public abstract string GetReference(object value, out bool alreadyExists);
+        public abstract object ResolveReference(string referenceId);
+    }
+}
+
+namespace System.Text.Json.Serialization.Metadata
+{
+    public partial class DefaultJsonTypeInfoResolver : IJsonTypeInfoResolver
+    {
+        public Collections.Generic.IList<Action<JsonTypeInfo>> Modifiers { get { throw null; } }
+
+        public virtual JsonTypeInfo GetTypeInfo(Type type, JsonSerializerOptions options) { throw null; }
+    }
+
+    public partial interface IJsonTypeInfoResolver
+    {
+        JsonTypeInfo? GetTypeInfo(Type type, JsonSerializerOptions options);
+    }
+
+    public sealed partial class JsonCollectionInfoValues<TCollection>
+    {
+        public JsonTypeInfo ElementInfo { get { throw null; } init { } }
```

Comment:
The `init` is incorrectly generated here and should be changed to `set`. See https://github.com/dotnet/source-build-reference-packages/pull/861#issuecomment-1882031209.

Reply:
Is there a plan to update the tooling to handle this? This is already far too much friction for updating a single version number and it's only going to get worse as the push to get these updated asap ramps up.

Reply:
Also, the generate command when I ran it generated a bunch of additional files that I had to mentally filter out for other packages (System.Text.Encodings.Web, System.Runtime.InteropServices, and Microsoft.Bcl.asyncingterfaces). It'd be great to clean up the generation tool for that as well because if we're going to enable nuget audit and be updating packages regularly that aren't in source build, we need the process to be as smooth as possible.

Reply:
> Also, the generate command when I ran it generated a bunch of additional files that I had to mentally filter out for other packages (System.Text.Encodings.Web, System.Runtime.InteropServices, and Microsoft.Bcl.asyncingterfaces). It'd be great to clean up the generation tool for that as well because if we're going to enable nuget audit and be updating packages regularly that aren't in source build, we need the process to be as smooth as possible.

This is called out in https://github.com/dotnet/source-build-reference-packages?tab=readme-ov-file#workflow. It is ok to include those changes.